tests/modules/sharedaddy/test_class.recaptcha.php

```diff
@@ -0,0 +1,139 @@
+<?php
+require dirname( __FILE__ ) . '/../../../modules/sharedaddy/recaptcha.php';
+
+class WP_Test_Jetpack_ReCaptcha extends WP_UnitTestCase {
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->site_key   = defined( 'RECAPTCHA_PUBLIC_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'sitekey';
+		$this->secret_key = defined( 'RECAPTCHA_PRIVATE_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'secretkey';
```

Comment:
Minor: It might be better to use our own mock values rather than the actual constants since we're mocking the requests.